packages/server/modules/cross-server-sync/services/project.ts

```diff
@@ -159,6 +167,18 @@ const importVersions = async (params: {
       `/projects/${projectId}/models/${version.model.id}@${version.id}`,
       origin
     )
+
+    const downloadCommit = downloadCommitFactory({
```

Comment:
I'm assuming I'm supposed to then refactor this part of the module that invokes the factory as a separate PR